chatmaild/pyproject.toml

```diff
@@ -13,6 +13,7 @@ dependencies = [
   "filelock",
   "requests",
   "crypt-r >= 3.13.1 ; python_version >= '3.11'",
+  "pytest-localserver",
```

Comment:
This should go into `testenv` section of tox.ini, next to `pytest`, at the end of this file. We don't need pytest-localserver installed on the server, especially without pytest itself.